src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -26,25 +28,52 @@
       :accessibility-label :import-private-key
       :label               (i18n/label :t/import-private-key)}]]])
 
-(def accounts
-  [{:account-props {:customization-color :turquoise
-                    :size                32
-                    :emoji               "\uD83C\uDFB2"
-                    :type                :default
-                    :name                "Trip to Vegas"
-                    :address             "0x0ah...71a"}
-    :networks      [{:network-name :ethereum :short-name "eth"}
-                    {:network-name :optimism :short-name "opt"}]
-    :state         :default
-    :action        :none}])
+(defn parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [colorId emoji name address]}]
```

Comment:
hmm, I think we should update this data properly as colorId looks the status-go api, however we usually defined this as `color` or `customization-color`. it needs to be handled in a "data-store" type approach, right @smohamedjavid ?

Reply:
Right, @J-Son89.

Reply:
@J-Son89 fixed!